docs/flatcar-linux/packet.md

```diff
@@ -326,3 +326,13 @@ Reference the DNS zone id with `"${aws_route53_zone.zone-for-clusters.zone_id}"`
 | reservation_ids_default | Default hardware reservation ID for nodes not listed in the `reservation_ids` map. | "" | "next-available"|
 
 Documentation about Packet hardware reservation id can be found here: https://support.packet.com/kb/articles/reserved-hardware.
+
+## Post-installation modification
+
+Currently the only tested ways to modify a cluster are:
+
+* Adding new worker pools, done by adding a new worker module and concating its `.worker_nodes_hostname` to the `worker_nodes_hostnames` list of the controller module
+* Scaling a worker pool by changing the `count` to delete or add nodes, even to 0 (but the worker pool definition has to be kept)
+* Changing the instance type of a worker pool by altering `type`, e.g., from `t1.small.x86` to `c1.small.x86`, which will recreate the nodes
```

Comment:
I'll be more explicit that nodes won't be recreated in any order, they will be destroyed first and then created again so it may create downtime.

Reply:
Good point.